### DISCUSSION TITLE
JSON Schema here to help

### DISCUSSION THREAD

**Discussion by Relequestual** [ehn-digital-green-development/ehn-dgc-schema#95] [General]
Hello!

Ben Hutton here from the JSON Schema team.

A link to this repo came across my desk.
Please let us know if we can be of any help or assistance. We are on hand.

If you would like to schedule a call, that's also possible.

Great to see you're using the latest draft of JSON Schema. We worked hard to make it happen.

**Comment by Pat7584** [comment]
Hi 
Thank you for welcome me
I need some help
I use windows 10, Visual studio
I have some trouble with the make, i obtain
Checking JSON formatting...
DCC.Core.Types.schema.json DCC.Core.Types.schema.json.tmp differ: byte 2, line 1
Please reformat DCC.Core.Types.schema.json
DCC.schema.json DCC.schema.json.tmp differ: byte 2, line 1
Please reformat DCC.schema.json
DCC.Types.schema.json DCC.Types.schema.json.tmp differ: byte 2, line 1
Please reformat DCC.Types.schema.json
DCC.ValueSets.schema.json DCC.ValueSets.schema.json.tmp differ: byte 2, line 1
and so on
and at the end
/node_modules/.bin/ajv -c ajv-formats --spec=draft2020 --strict=false test -s DCC.combined-schema.json -d "examples/vaccination/*.json" --valid
error:  C:\Users\patri\Documents\ehn-dcc-schema-release-1.3.0 (1)\ehn-dcc-schema-release-1.3.0\DCC.combined-schema.json: Unexpected end of JSON input
make: *** [Makefile:23: test] Error 2

Thank you for any help

**Reply by Relequestual** [reply]
You may want to create a new discussion or issue. In my original message, I was addressing the repository maitainers.

**Comment by dslmeinte** [comment]
> Hello!
> 
> Ben Hutton here from the JSON Schema team.
> 
> A link to this repo came across my desk.
> Please let us know if we can be of any help or assistance. We are on hand.
> 
> If you would like to schedule a call, that's also possible.
> 
> Great to see you're using the latest draft of JSON Schema. We worked hard to make it happen.

Hi Ben,

Apparently, we've managed to overlook your message entirely for the last 2 years: sorry about that!
Thanks for your hard work on JSON Schema, which evidently is of such quality that we didn't require any assistance ;)

**Comment by dslmeinte** [comment]
Hi Pat,

Sorry to have missed this message. Indeed, as Ben pointed out: this {c|sh}ould have been an Issue.
